release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -4066,3 +4066,31 @@ $ oc adm release info 4.12.33 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-12-34"]
+=== RHBA-2023:5151 - {product-title} 4.12.34 bug fix update
+
+Issued: 2023-09-20
+
+{product-title} release 4.12.34 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:5151[RHBA-2023:5151] advisory. There are no RPM packages for this release.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.12.34 --pullspecs
+----
+
+[id="ocp-4-12-34-bug-fixes"]
+==== Bug fixes
+
+* Previously, a non-compliant upstream DNS server that provided a UDP response larger than {product-title} specified bufsize of 512 bytes, caused an overflow error in CoreDNS in which a response to a DNS query was not given. With this update, users can configure the `protocolStrategy` field on the `dnses.operator.openshift.io` custom resource to be "TCP". This resolves issues with non-compliant upstream DNS servers. (link:https://issues.redhat.com/browse/OCPBUGS-15251[OCPBUGS-15251])
```

Comment:
Shouldn't these OCPBUGS links be in bold? It looks like they were bolded up until 4.12.29, which I'm assuming you copied this from. So maybe consider fixing in both places?